java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -379,6 +379,29 @@ public byte getByte(long index) {
     return offHeap.data.getByte(index * type.sizeInBytes);
   }
 
+  /**
+   * Get the byte list at index.
+   */
+  public byte[] getByteList(long index) {
+    assert type == DType.LIST;
+    assertsForGet(index);
+
+    int start = offHeap.offsets.getInt(index * DType.INT32.getSizeInBytes());
+    int end = offHeap.offsets.getInt((index + 1) * DType.INT32.getSizeInBytes());
+    int size = end-start;
+
+    // check if null or empty
+    if (start == end) {
```

Comment:
This is not guaranteed.  Technically in arrow it is possible to have a null value where the size  > 0.

Reply:
That is informative. This is present in  getList call as well which can be removed and replaced with just 
```
if (isNull(rowIndex)) {
        return null;
      }
```